Mathlib/NumberTheory/Divisors.lean

```diff
@@ -43,8 +45,12 @@ def divisors : Finset ℕ := {d ∈ Ico 1 (n + 1) | d ∣ n}
   As a special case, `properDivisors 0 = ∅`. -/
 def properDivisors : Finset ℕ := {d ∈ Ico 1 n | d ∣ n}
 
-/-- `divisorsAntidiagonal n` is the `Finset` of pairs `(x,y)` such that `x * y = n`.
-  As a special case, `divisorsAntidiagonal 0 = ∅`. -/
+/-- Pairs of divisors of a natural number as a finset.
+
+`n.divisorsAntidiagonal` is the finset of pairs `(a, b) : ℕ × ℕ` such that `a * b = n`.
+As a special case, `Nat.divisorsAntidiagonal 0 = ∅`.
+
+O(n). -/
```

Comment:
I put it to you that "$$O(n)$$." is not a full sentence.

Reply:
I am following the convention established by https://leanprover-community.github.io/mathlib4_docs/Init/Data/List/Basic.html#List.range